### PR TITLE
Use pytest.ini to specify `--color=yes`

### PR DESCRIPTION
**Pull request by harupy**
## What changes are proposed in this pull request?

Instead of adding `--color=yes` to all the pytest commands, we can write it in `pytest.ini`.

## How is this patch tested?

Verified the pytest output is colored.

https://github.com/mlflow/mlflow/pull/2716/checks?check_run_id=592800991

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Model Registry
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --color=yes
```
Comment (harupy) [inline pytest.ini:2]: With this configuration, `pytest` is equivalent to `pytest --color=yes`.

https://docs.pytest.org/en/latest/customize.html#adding-default-options

Comment (harupy) [reply pytest.ini:2]: It seems all the pytest commands contain `--verbose`. Should it also be included in `addopts`?

Comment (aarondav) [reply pytest.ini:2]: I guess the small test runner doesn't have `--verbose`, so can probably just keep this here.

**Comment by aarondav** [comment]
@harupy Thanks for the change! By the way, Mani's PR went in which moved around the large tests, mind rebasing for that? #2710

**Review by aarondav — APPROVED** [review]

**Comment by aarondav** [comment]
LGTM! Will merge after tests pass.

**Pull request merged**